File: 005/005.rb
number = 1
result = 1
i = 1
loop do
  i = 20
  while i > 1 do
    result = number % i
    i = i - 1
    break if result > 0
  end
  break if result == 0
  number += 1
end
puts number

# this is a very slow solution! takes about 8-9 minutes
# can be made faster with the use of rational methods.

File: 005/005.c
int main(void)
{
	int number = 1;
	int result = 1;
	int i = 1;
	
	while(result != 0 )
	{	
		number++;
		i = 20;
		while( i > 1 )
		{
			result = number % i;
			i--;
			if(result > 0)
			{
				break;
			}
		}
	}
	printf("%d", number);
	return 1;
}